pkg/asset/installconfig/aws/basedomain.go

```diff
@@ -22,28 +22,33 @@ func IsForbidden(err error) bool {
 
 // GetBaseDomain returns a base domain chosen from among the account's
 // public routes.
```

Comment:
Add something like:

> Also returns the zone ID for the chosen base domain, to distinguish between multiple public zones that share the same base domain.

And maybe use named variables in the return:

```go
func GetBaseDomain() (baseDomain string, zoneID string, err error) {
```

Reply:
Will do. Thanks for the heads up about named parameters!